test/harry/main/org/apache/cassandra/harry/model/QuiescentChecker.java

```diff
@@ -208,17 +209,20 @@ public static void validate(SchemaSpec schema, DataTracker tracker, @Nullable Se
                 Reconciler.RowState expectedStaticRowState = adjustForSelection(partitionState.staticRow(), schema, selection, true);
                 assertStaticRow(partitionState, actualRows, expectedStaticRowState, actualRowState, query, trackerState, schema);
             }
+
+            rowId++;
```

Comment:
I added this as it wasn't clear that C* returned 0 rows... with this I can tell the difference 0 rows and missing at least 1 row